veros/setup/global_4deg/global_four_degree.py

```diff
@@ -119,6 +124,7 @@ def set_coriolis(self, vs):
     def set_topography(self, vs):
         bathymetry_data = self._read_forcing(vs, 'bathymetry')
         salt_data = self._read_forcing(vs, 'salinity')[:, :, ::-1]
+        salt_data[vs.zt[np.newaxis, np.newaxis, :] <= bathymetry_data[..., np.newaxis]] = 0.
```

Comment:
Let's absorb this into `mask_bathy`:

```python
mask_bathy = vs.zt[np.newaxis, np.newaxis, :] <= bathymetry_data[..., np.newaxis]
```

Makes is a bit clearer what's going on.